src/Behaviour/HtmlCrawlerManagerInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the "StenopePHP/Stenope" bundle.
+ *
+ * @author Thomas Jarrand <thomas.jarrand@gmail.com>
+ */
+
+namespace Stenope\Bundle\Behaviour;
+
+use Stenope\Bundle\Content;
+use Symfony\Component\DomCrawler\Crawler;
+
+interface HtmlCrawlerManagerInterface
+{
+    /**
+     * Get HTML Crawler for the given property (creats it if needed)
+     */
+    public function get(Content $content, array &$data, string $property): ?Crawler;
```

Comment:
```suggestion
    public function get(Content $content, array $data, string $property): ?Crawler;
```

(it would be weird for an implementation to mutate the data when calling `get`, right?